tests/unit/backend/corpora/api_server/curator/collection/test_curation_collections.py

```diff
@@ -128,6 +128,61 @@ def test__create_collection__InvalidParameters(self):
                     self.assertIn(error, response.json["detail"])
 
 
+class TestGetCollections(BaseAuthAPITest):
+    def setUp(self):
+        super().setUp()
+        self.test_collection = dict(
+            name="collection", description="description", contact_name="john doe", contact_email="johndoe@email.com"
+        )
+
+    def test__get_collections_no_auth__OK(self):
+        res_no_auth = self.app.get("/curation/v1/collections")
+        self.assertEqual(200, res_no_auth.status_code)
+        self.assertEqual(6, len(res_no_auth.json["collections"]))
+        [self.assertEqual("PUBLIC", c["visibility"]) for c in res_no_auth.json["collections"]]
+
+    def test__get_collections_no_auth_processing_status_PENDING__OK(self):
+        res_no_auth = self.app.get("/curation/v1/collections")
+        self.assertEqual(200, res_no_auth.status_code)
+        self.assertEqual(6, len(res_no_auth.json["collections"]))
+        for collection in res_no_auth.json["collections"]:
+            if collection["id"] == "test_collection_id":
+                self.assertEqual(collection["processing_status"], "PENDING")
+            else:
+                self.assertEqual(collection["processing_status"], "SUCCESS")
+        [self.assertEqual("PUBLIC", c["visibility"]) for c in res_no_auth.json["collections"]]
```

Comment:
We test this same condition on the same data in `test__get_collections_no_auth__OK`. Is there enough difference between the two tests to require two separate tests?

Reply:
Addressed by combining with subTest